### PR TITLE
Changed ini syntax highlighting to sql syntax highlighting and remove…

### PR DESCRIPTION
**Pull request by jpe442**
Removing  pem =# from code blocks and changing ini syntax highlighting to sql.

### CODE REVIEW

File: product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx [hunk 1]
```diff
@@ -11,48 +11,48 @@ You must configure the PEM database server to work with PgBouncer. This example
 
 1.  Create a dedicated user named pgbouncer on the PEM database server:
 
-    ```ini
-    pem=# CREATE USER pgbouncer PASSWORD 'ANY_PASSWORD' LOGIN;
+    ```sql
+    CREATE USER pgbouncer PASSWORD 'ANY_PASSWORD' LOGIN;
     CREATE ROLE
```
Comment (djw-m) [inline product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx:16]: Create role is output and should be preceded by __OUTPUT__ in the code block to style it correctly.

File: product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx [hunk 2]
```diff
@@ -11,48 +11,48 @@ You must configure the PEM database server to work with PgBouncer. This example
...
     ```
 
 2.  Create a user named pem_admin1 (not a superuser) with `pem_admin` and `pem_agent_pool role` membership on the PEM database server:
 
-    ```ini
-    pem=# CREATE USER pem_admin1 PASSWORD 'ANY_PASSWORD' LOGIN
+    ```sql
+    CREATE USER pem_admin1 PASSWORD 'ANY_PASSWORD' LOGIN
     CREATEROLE;
```
Comment (djw-m) [inline product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx:23]: CREATEROLE looks like it should be on the same line as the command - let the code block wrapper take the strain.

File: product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx [hunk 3]
```diff
@@ -11,48 +11,48 @@ You must configure the PEM database server to work with PgBouncer. This example
...
     CREATE ROLE
```
Comment (djw-m) [inline product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx:24]: And again __OUTPUT__

File: product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx [hunk 4]
```diff
@@ -11,48 +11,48 @@ You must configure the PEM database server to work with PgBouncer. This example
...
-    pem=# GRANT pem_admin, pem_agent_pool TO pem_admin1;
+    GRANT pem_admin, pem_agent_pool TO pem_admin1;
```
Comment (djw-m) [inline product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx:25]: This is input again, so I'd suggest breaking up the code block and popping in a new __OUTPUT__

File: product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx (hunk 1 above)
Comment (jpe442) [reply product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx:17]: got it 👍

File: product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx (hunk 2 above)
Comment (jpe442) [reply product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx]: got it 👍

File: product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx (hunk 3 above)
Comment (jpe442) [reply product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx:25]: got it 👍

File: product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx (hunk 4 above)
Comment (jpe442) [reply product_docs/docs/pem/8/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx:28]: got it

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**